### PR TITLE
Add breadcrumbs to Pages

### PR DESCRIPTION
**Pull request by EdwinKruglov**
## Changes in this PR

- add `breadcrumbs` field to `pages`
- add a method to generate breadcrumbs based on `Page.breadcrumbs`

## Screenshots of UI changes

image

### CODE REVIEW

**Comment by EdwinKruglov** [comment]
[Add breadcrumbs to Pages](https://trello.com/c/hQTcrHRL/109-add-breadcrumbs-to-pages)

File: app/controllers/pages_controller.rb [hunk 1]
```diff
@@ -21,4 +25,13 @@ def self.bypass_dsi?
   def page
     @page ||= Page.find_by(slug: params[:slug])
   end
+
+  def set_breadcrumbs(page)
+    if page.breadcrumbs.present?
+      page.breadcrumbs.each do |item|
+        title, path = item.split(",")
+        breadcrumb title, path
```
Comment (peterdavidhamilton) [inline app/controllers/pages_controller.rb:30]: `breadcrumb *item.split(",")` is an alternative. What does a breadcrumb in Contentful look like? Can we document it here

**Review by d-a-v-e — APPROVED** [review]

File: app/controllers/pages_controller.rb [hunk 2]
```diff
@@ -21,4 +22,13 @@ def self.bypass_dsi?
   def page
     @page ||= Page.find_by(slug: params[:slug])
   end
+
+  def set_breadcrumbs
+    return if page.blank?
```
Comment (peterdavidhamilton) [inline app/controllers/pages_controller.rb:27]: Maybe `page&.breadcrumbs`

**Comment by peterdavidhamilton** [comment]
Don't forget the screenshots

**Review by PiperS52 — APPROVED** [review]

**Review by peterdavidhamilton — APPROVED** [review]

**Pull request merged**